### DISCUSSION TITLE
for what is "site_address" and "trusted_host_patterns" useful?

### DISCUSSION THREAD

**Discussion by comsyspro** [LiveHelperChat/livehelperchat#1886] [General]
in the docs there are described the settings "site_address" and "trusted_host_patterns" see: https://doc.livehelperchat.com/docs/security/#trusted-hosts-and-site-address

what exactly do these settings? are they important for security and why should you set them?

is it also possible to prevent the direct access of this static link, so it is instead only possible to acces this over the widget script or could you hide at least the department option selection there? see: https://demo.livehelperchat.com/chat/start

**Comment by remdex** [comment]
It defines what domains/subdomains can embed a widget.

To hide department and avoid directly URL see
https://doc.livehelperchat.com/docs/chat/start-chat-form-settings#disable-start-chat-url
https://doc.livehelperchat.com/docs/department/#how-to-disable-user-to-change-department
https://doc.livehelperchat.com/docs/javascript-arguments#how-do-i-set-default-url-for-live-helper-chat

**Reply by comsyspro** [reply]
`It defines what domains/subdomains can embed a widget.` for that there is also a system settings in `Home / Chat configuration -> Domains where script can be embedded. E.g example.com, google.com`. is there a difference between these two settings? or do you need to set both or only one setting is enough?

**Reply by remdex** [reply]
Setup both.
